rebar.config

```diff
@@ -56,6 +56,7 @@
    [{deps, [{ranch, "2.0.0"}]}]},
   {test, 
    [
+    {erl_opts, [nowarn_export_all]},
```

Comment:
Could you, please, just replace `-compile([export_all]).` with `-compile([export_all, nowarn_export_all]).` in test module's code instead of disabling this warning for all the modules?

Reply:
Sure. It's only for "all modules in profile `test`" but I can do that.

Reply:
06be1b8